cranelift/filetests/filetests/isa/aarch64/stack-limit.clif

```diff
@@ -43,7 +43,7 @@ block0(v0: i64):
 ;   b.hs 8 ; udf
 ; block0:
 ;   ldr x2, 8 ; b 12 ; data TestCase(%foo) + 0
-;   blr x2
+;   blr x2 # CallIndInfo { rn: v129, uses: [], defs: [], clobbers: PRegSet { bits: 79228162495817593519834660863 }, opcode: Call, caller_callconv: Fast, callee_callconv: Fast }
```

Comment:
Kinda strange to have two different kinds of comments here? Should `#` just be `;`?

Reply:
`;` is actually not a comment in the generated assembly format, though -- the punctuation above breaks down as:

- initial column `;` is `filecheck`'s comment, and serves here to give the `precise-output` expected text
- the `;`s in line 45 separate additional instructions (`ldr`, then `b`, then a `data` pseudoinstruction)
- the `#` on line 46 introduces an actual comment that gives more info about the call

so the `; ... # ...` is a comment-within-a-comment, if that makes sense...